ff/serving/provider/sql.go

```diff
@@ -145,12 +144,19 @@ func (store *sqlOfflineStore) tableExists(id ResourceID) (bool, error) {
 	}
 	query := store.query.tableExists()
 	err = store.db.QueryRow(query, tableName).Scan(&n)
-	if n == 0 {
-		return false, nil
+	if n > 0 && err == nil {
+		return true, nil
 	} else if err != nil {
```

Comment:
no need for an else if since the previous is a return

Reply:
yeah could change that to two IFs, but felt like theres more organization with if/elseif

Reply:
ok just a style thing no biggie